juju/utils.py

```diff
@@ -20,11 +22,11 @@
 
 async def execute_process(*cmd, log=None):
     """Wrapper around asyncio.create_subprocess_exec."""
-    p = await jasyncio.create_subprocess_exec(
+    p = await asyncio.create_subprocess_exec(
         *cmd,
-        stdin=jasyncio.subprocess.PIPE,
-        stdout=jasyncio.subprocess.PIPE,
-        stderr=jasyncio.subprocess.PIPE,
+        stdin=asyncio.subprocess.PIPE,
+        stdout=asyncio.subprocess.PIPE,
+        stderr=asyncio.subprocess.PIPE,
```

Comment:
why the change to not using `jasyncio` here? I notice that it's still imported (line 18)

Reply:
this was general cleanup, though I'm not doing the bulk 
 of it, leaving that for #1178